src/util/sharing_map.h

```diff
@@ -54,6 +54,16 @@ Author: Daniel Poetzl
     typename equalT> \
   type sharing_mapt<keyT, valueT, fail_if_equal, hashT, equalT>
 
+#define SHARING_MAPTV(return_type, V) \
```

Comment:
Given that this is used in exactly one place I’d like to voice a strong preference for just inlining this macro.